### PR TITLE
Add ADVA FSP150CM and associated modules

### PR DESCRIPTION
**Pull request by dar72**
Adding the ADVA FSP150CM along with NEMI and NTU-GE modules

### CODE REVIEW

**Comment by danner26** [comment]
Hello, I have fixed the formatting so that the tests pass. Is this PR ready to be reviewed?

**Comment by dar72** [comment]
Why does

> Hello, I have fixed the formatting so that the tests pass. Is this PR ready to be reviewed?

Thankyou, what was causing it to fail?

Yes, it's ready

**Comment by danner26** [comment]
The issue was due to spacing

File: device-types/ADVA/FSP-150-CM.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
+---
+manufacturer: ADVA
+model: FSP-150-CM
+slug: adva-fsp-150-cm
+part_number: '1044004001'
+u_height: 4
+is_full_depth: false
```
Comment (danner26) [inline device-types/ADVA/FSP-150-CM.yaml:7]: Please add the `airflow` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

Comment (danner26) [inline device-types/ADVA/FSP-150-CM.yaml:7]: Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

File: device-types/ADVA/FSP-150-CM.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+interfaces:
+  - name: BITSI/O
+    type: 1000base-t
+    mgmt_only: true
+  - name: SER
+    type: 1000base-t
+    mgmt_only: true
+  - name: DCN1
+    type: 10gbase-x-sfpp
+    mgmt_only: false
```
Comment (danner26) [inline device-types/ADVA/FSP-150-CM.yaml:27]: Please remove the `mgmt_only: false` definition

File: device-types/ADVA/FSP-150-CM.yaml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+manufacturer: ADVA
+model: FSP-150-CM
+slug: adva-fsp-150-cm
+part_number: '1044004001'
+u_height: 4
+is_full_depth: false
+console-ports:
+  - name: rs-232
+    type: rj-45
+power-ports:
```
Comment (danner26) [inline device-types/ADVA/FSP-150-CM.yaml:11]: Are the power supplies on these modular? If so, then the PSUs should be created under `module-bays`, and the associated PSUs should also be created in the module-types folder if they do not already exist

**Review by danner26 — CHANGES_REQUESTED** [review]

File: device-types/ADVA/FSP-150-CM.yaml (hunk 1 above)
Comment (dar72) [reply device-types/ADVA/FSP-150-CM.yaml:7]: I can't find the datasheet, these are older models and ADVA seem to have removed the datasheets from their website

File: device-types/ADVA/FSP-150-CM.yaml (hunk 3 above)
Comment (dar72) [reply device-types/ADVA/FSP-150-CM.yaml]: Done. Again, I can't find a datasheet for any of this so I've given it a sensible name and model

File: module-types/ADVA/NTU-GE.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+---
+manufacturer: ADVA
+model: NTU-GE
+part_number: '1044441001'
```
[the remark this answers is not in the document]
Comment (dar72) [reply module-types/ADVA/NTU-GE.yaml:4]: No datasheet for this either

File: module-types/ADVA/NEMI.yaml [hunk 5]
```diff
@@ -0,0 +1,10 @@
+---
+manufacturer: ADVA
+model: NEMI
+part_number: 1044200020-05
```
[the remark this answers is not in the document]
Comment (dar72) [reply module-types/ADVA/NEMI.yaml:4]: No datasheet

**Pull request merged**